test/runtests.jl

```diff
@@ -1306,6 +1310,10 @@ end
         @test record.tlen == new_record.tlen
         @test record.data == new_record.data
 
+        # rightposition (also implicitly alignlength)
+        records = collect(open(BAM.Reader, joinpath(bamdir, "ce#5b.bam")))
+        @test BAM.rightposition(records[6]) == rightposition(records[6]) == 83
```

Comment:
Did you add this same code twice? It's [also here](https://github.com/BioJulia/BioAlignments.jl/pull/38/files#diff-fce720c43af3c52c862fd7451c7374b8R1152-R1155)

Reply:
Oh, I see it's sam and bam, nevermind!